test/e2e/kam_test.go

```diff
@@ -19,7 +19,7 @@ func validateKamService(t *testing.T) {
 	framework.AddToFrameworkScheme(routev1.AddToScheme, &routev1.Route{})
 	framework.AddToFrameworkScheme(console.AddToScheme, &console.ConsoleCLIDownload{})
 
-	ctx := framework.NewTestCtx(t)
+	ctx := framework.NewContext(t)
```

Comment:
If you are going to replace them, please replace all `NewTextCtx`

Reply:
Done